pybamm/models/lead_acid/newman_tiedemann.py

```diff
@@ -0,0 +1,125 @@
+#
+# Lead-acid Porous-Electrode model
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pybamm
+
+
+class NewmanTiedemann(pybamm.LeadAcidBaseModel):
+    """Porous electrode model for lead-acid, from [1]_.
+
+    .. math::
+        \\frac{\\partial \\tilde{c}}{\\partial t}
+        = \\frac{1}{\\varepsilon^{(0)}}\\left(
+            \\frac{D^{\\text{eff}, (0)}}{\\mathcal{C}_\\text{e}}
+            \\frac{\\partial^2 \\tilde{c}}{\\partial x^2}
+            + \\left(
+                s + \\beta^{\\text{surf}}c^{(0)}
+            \\right)j^{(0)}
+        \\right)
+
+
+    **Notation for variables and parameters:**
+
+    * f_xy means :math:`f^{(x)}_\\text{y}` (x is the power for the asymptotic \
+    expansion and y is the domain). For example c_1n means :math:`c^{(1)}_\\text{n}`, \
+    the first-order concentration in the negative electrode
+    * fbar_n means :math:`\\bar{f}_n`, the average value of f in that domain, e.g.
+
+    .. math::
+        \\text{cbar_n}
+        = \\bar{c}_\\text{n}
+        = \\frac{1}{\\ell_\\text{n}}
+        \\int_0^{\\ell_\\text{n}} \\! c_\\text{n} \\, \\mathrm{d}x
+
+    **Extends:** :class:`pybamm.LeadAcidBaseModel`
+
+    References
+    ==========
+    .. [1] V Sulzer, SJ Chapman, CP Please, DA Howey, and CW Monroe. Faster Lead-Acid
+           Battery Simulations from Porous-Electrode Theory: I. Physical Model.
+           arXiv preprint arXiv:1902.01771, 2019.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        # Parameters
+        param = pybamm.standard_parameters
+        param.__dict__.update(pybamm.standard_parameters_lead_acid.__dict__)
+
+        #
+        # Variables
+        #
+        whole_cell = ["negative electrode", "separator", "positive electrode"]
+        #
+        # Variables
+        #
+        # Concentration variables
+        c_e_n = pybamm.Variable(
+            "Negative electrode concentration", domain=["negative electrode"]
```

Comment:
Might be slightly clearer to call this "electrolyte concentration in negative electrode". That way it is clear that its in the electrolyte and not the electrode. There isn't any confusion in lead-acid but there might be in li-ion.